app/Contributor/ContributorServiceProvider.php

```diff
@@ -27,15 +26,9 @@ public function register(): void
                 new DbalContributorRepository($container->get('dbal_connection')),
                 $container->get(EventBus::class),
                 new ContributorsUpdatedFactory(
```

Comment:
The name for this factory is a bit weird now `ContributorsUpdatedFactory` because those messages no longer exist.

Reply:
Maybe there is already another similar factory for this, now that we return the existing `ProjectedToJSONLD` messages?

Reply:
There does not seem to be a Generic ProjectedToJSONLDFactory, maybe now is a good time to create one?

Reply:
Sounds good!